### PR TITLE
VULCAN-166: excel export

### PR DESCRIPTION
**Pull request by timwongj**
Export projects to excel with each component being a tab

Closes: https://github.com/mitre/vulcan/issues/166

### CODE REVIEW

**Comment by rbclark** [comment]
@timwongj Currently failing on Rubocop

**Comment by timwongj** [comment]
> @timwongj Currently failing on Rubocop

Fixed

File: app/javascript/components/project/Project.vue [hunk 1]
```diff
@@ -248,6 +255,17 @@ export default {
...
+      axios
+        .get(`/projects/${this.project.id}/export/excel`)
+        .then((response) => {
+          FileDownload(
+            base64StringToBlob(response.data.replace(/(\r\n|\n|\r)/gm, ""), "application/xlsx"),
+            `${this.project.name}.xlsx`
+          );
+        })
+        .catch(this.alertOrNotifyResponse);
+    },
```
Comment (rbclark) [inline app/javascript/components/project/Project.vue:268]: It seems the download method used for downloading this versus the one used for downloading component exports are very different, could these be consolidated on a single download method?

Comment (timwongj) [reply app/javascript/components/project/Project.vue]: done

File: app/javascript/components/project/Project.vue [hunk 2]
```diff
@@ -121,6 +130,8 @@
 <script>
 import _ from "lodash";
 import axios from "axios";
+import FileDownload from "js-file-download";
+import base64StringToBlob from "base64toblob";
```
Comment (rbclark) [inline app/javascript/components/project/Project.vue:134]: Can this dependency be removed here and in the package.json?

Comment (timwongj) [reply app/javascript/components/project/Project.vue:134]: good call

Comment (timwongj) [reply app/javascript/components/project/Project.vue]: done

**Review by rbclark — APPROVED** [review]

**Pull request merged**